### PR TITLE
Format html using only the plugin itself

### LINKED ISSUE #75: 🐛 BUG: Header and Footer components are being lowercased
### Describe the Bug

When I use a component `<Header />` or `<Footer />` (maybe happens with more components, but I'm not aware of that) they are converted to `<header />` and `<footer />` HTML elements when they shouldn't

### Steps to Reproduce

1. `npm init astro` using template `minimal`
2. Install `prettier` and `prettier-plugin-astro`
3. Create two components: `Header.astro` and `Footer.astro`
4. Replace your `index.astro` with this:
```astro
---
import Header from "../components/Header.astro";
import Footer from "../components/Footer.astro";
---

<html lang="en">
  <body>
    <Header />

    <h1>Welcome to <a href="https://astro.build/">Astro</a></h1>

    <Footer />
  </body>
</html>
```
5. Run `prettier -w .` (maybe you'll have to run it twice)
6. You'll see the components `<Header />` to be `<header />` (and the same with Footer)

### PR DESCRIPTION
**Pull request by antonyfaris**
## Changes

The plugin will always format HTML instead of conditionally.
Fixes #75 

## Testing

I change some tests see the comments.

### CODE REVIEW

File: test/fixtures/embedded-in-markdown/output.md [hunk 1]
```diff
@@ -59,9 +59,7 @@ For best results, you should only have one `<style>` tag per-Astro component. Th
       ...;
     </style>
   </head>
-  <body>
-    ...
-  </body>
+  <body> ... </body>
```
Comment (antonyfaris) [inline test/fixtures/embedded-in-markdown/output.md:62]: We need to implement the logic for when the content of an element needs to break.

File: test/fixtures/embedded-in-markdown/output.md [hunk 2]
```diff
@@ -261,9 +259,9 @@ An Astro component template can render as many top-level elements as you'd like.
 
 ```astro
 <!-- An Astro component can contain multiple top-level HTML elements: -->
-<div id="a" />
-<div id="b" />
-<div id="c" />
+<div id="a"></div>
```
Comment (antonyfaris) [inline test/fixtures/embedded-in-markdown/output.md:262]: We discuss this in discord and it was the preferred output.

File: test/fixtures/option-html-whitespace-sensitivity-ignore/input.astro [hunk 3]
```diff
@@ -5,5 +5,4 @@
 <div class="voluptatem architecto at 1">Architecto rerum architecto incidunt sint.</div>
 <div class="voluptatem architecto at 2">  Architecto rerum architecto incidunt sint.</div>
 <div class="voluptatem architecto at 3">Architecto rerum architecto incidunt sint.  </div>
- <div class="voluptatem architecto at 4">  Architecto rerum architecto incidunt sint.  </div>
- <p>lorem</p><p>lorem</p><span>lorem</span><span>lorem</span>
```
Comment (antonyfaris) [inline test/fixtures/option-html-whitespace-sensitivity-ignore/input.astro:9]: This was left accidentally by me in another pr.

File: test/fixtures/preserve-tag-case/input.astro [hunk 4]
```diff
@@ -3,6 +3,8 @@ import Footer from "./Footer.astro";
 import  {Body}  from "lib";
 ---
 
+  <div>
```
Comment (antonyfaris) [inline test/fixtures/preserve-tag-case/input.astro:6]: I add this so can be sure is working as excepted.

**Review by jasikpark — APPROVED** [review]
lgtm

**Pull request merged**